samples/authorization/certificate_v200-alpha.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
```

Comment:
This file can be removed. Self-signed certificates get created automatically if certificate and privateKey are empty strings in the CR

Reply:
Fixed